src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js

```diff
@@ -0,0 +1,75 @@
+import dateRangeUI from 'us-forms-system/lib/js/definitions/dateRange';
+
+import HospitalizationPeriodView from '../components/HospitalizationPeriodView';
+
+export const uiSchema = {
+  hospitalizationHistory: {
+    'ui:title': 'Hospitalization',
+    'ui:description': 'Dates you were hospitalized?',
+    'ui:options': {
+      itemName: 'Hospital',
+      viewField: HospitalizationPeriodView,
+      hideTitle: true,
+    },
+    items: {
+      hospitalizationtDateRange: dateRangeUI(
+        'From',
+        'To',
+        'End of hospitalization must be after start of treatment',
+      ),
+      hospitalName: {
+        'ui:title': 'Name of hospital',
+      },
+      streetAddress: {
```

Comment:
Is there anyway we can use the `addressUI()` here and omit the pieces we don't need(like Country).

Reply:
```
  hospitalAddress: {
            ...address,
            required: [],
            properties: omit(address.properties, ['addressLine3', 'country']),
          },
```
This is what I did. ^^